src/Mod/Draft/draftutils/init_draft_statusbar.py

```diff
@@ -0,0 +1,144 @@
+"""Draft Statusbar commands.
+
+This module provide the code for the Draft Statusbar, activated by initGui
+"""
+## @package init_tools
+# \ingroup DRAFT
+# \brief This module provides the code for the Draft Statusbar.
+
+# ***************************************************************************
+# *                                                                         *
+# *   This file is part of the FreeCAD CAx development system.              *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   FreeCAD is distributed in the hope that it will be useful,            *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with FreeCAD; if not, write to the Free Software        *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+import FreeCAD as App
+import FreeCADGui as Gui
+from PySide import QtGui
+from PySide.QtCore import QT_TRANSLATE_NOOP
+
+
+
+def set_draft_statusbar(show=True):
+    
+    draft_scales =  [QT_TRANSLATE_NOOP("draft","1:1000"),
```

Comment:
By the way, why are you translating these ratios? Do different languages treat this specially? Do French, Italian, Russian, etc. use other symbols for a ratio X:Y? I was not aware of this.

Reply:
I really don't know, but if they do they can translate it... I don't even know if Arabic numbers are used worldwide...

Reply:
Well, there exists of course other types of numerals like [Eastern Arabic numbers](https://en.wikipedia.org/wiki/Eastern_Arabic_numerals) and also [Chinese numbers](https://en.wikipedia.org/wiki/Chinese_numerals), but in general, everybody understands modern Indo-Arabic digits.

Reply:
Interesting! Well, it's the same to me... we can delete translations. No problem.

Reply:
As general rule, I would not translate any numbers or quantities, unless specifically requested by users.

Reply:
Agree with @vocx-fc ...

Reply:
ok, I'll change it.